tests/Aspire.StackExchange.Redis.DistributedCaching.Tests/Aspire.StackExchange.Redis.DistributedCaching.Tests.csproj

```diff
@@ -5,6 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Include="$(RepoRoot)src\Components\Aspire.StackExchange.Redis.DistributedCaching\ConfigurationSchema.json" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Why do we need to copy this file for the DistributedCaching and OutputCaching tests, if the file isn't going to be used?

I like that the base class will fail the test if the file can't be found. But it feels like there should be an "opt out" mechanism that these 2 tests can use to say "skip the ConfigurationSchema tests". Maybe a virtual boolean property on the base class that gets set in these 2 derived classes.

Reply:
> Why do we need to copy this file for the DistributedCaching and OutputCaching tests, if the file isn't going to be used?

My thinking was that we will at least be running `JsonSchema.FromFile` which would hopefully validate that the file is a valid xml/schema file.

Reply:
Ok, this makes sense. thanks.